src/TrackElementManager.jsx

```diff
@@ -81,15 +77,22 @@ export default class TrackElementManager extends React.Component {
             onCloseClick={this.onDeleteCloseClick.bind(this)}
             onConfirmClick={this.onDeleteConfirmClick.bind(this)} />
     </div>
-    
+
     <div className="clearfix"></div>
             </div>;
         } else {
             return <div></div>;
         }
     }
     onTextChange(e) {
-        this.props.onChange(this.props.activeElement, {source: e.target.value});
+        this.props.onChange(this.props.activeElement, {
+            source: e.target.value
+        });
+        jQuery('.jux-updated-text').show(0, function() {
```

Comment:
I see the UI flash that I think you're referring to - "Updated" flashes a few times when you make changes to the text field. I will address that, or we can change this field to instead update with an "OK" or Update button instead of updating immediately.